package.json

```diff
@@ -12,10 +12,12 @@
     "start": "node lib/index.js",
     "test": "jest",
     "test:watch": "npm test -- --watch",
-    "build": "npm run build:js && npm run build:sass",
+    "build": "npm run build:js && npm run build:js:error && npm run build:sass && npm run build:sass:error",
```

Comment:
Why are you redefining a new build process? Couldn't you merge into the existent one?

Reply:
Because I couldn't do it at the time. But maybe I didn't try hard enough.

Reply:
The problem is that the inputs scripts should be bundlize in separate files. That why I managed this issue like that.